third_party/islet/setup.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
```

Comment:
Please add a small banner prolog comment documenting what this shell script is for / about.

You may want to follow the convention introduced (by me) recently in `sample_apps/run_example.sh`.